src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.ts

```diff
@@ -40,6 +40,7 @@ export class CustomerListComponent implements OnInit, OnDestroy, AfterViewInit {
   showModal = false;
   idToDelete: string;
   idToEdit: string;
+  status: string;
```

Comment:
you could put a clearer name, since this status is the stutus of the item to be edited and now it seems to be a gloval status.

Reply:
Sure, working on it